### PR TITLE
docs: editing and translation for the APIServer doc #826

### LINKED ISSUE #826: editing and translation for the APIServer doc
This issue consists of two tasks about the documentation

- [ ] Proofreading the Chinese version of APIServer doc `docs/cn/apiserver.md`
- [ ] Translating this Chinese version to English, named as `docs/en/apiserver.md`

### PR DESCRIPTION
**Pull request by mahengyang**
Translating this Chinese version(docs/cn/apiserver.md) to English, named as docs/en/apiserver.md

### CODE REVIEW

File: docs/en/apiserver.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# APIServer
+
+To deploy `APIServer` click this: [Deploy OpenMLDB](deploy.md).
+
+Comparison table about JSON datatype and OpenMLDB datatype, and instructions of every function,[RFC](https://github.com/4paradigm/rfcs/blob/main/fedb/api-server.md).
```
Comment (lumianph) [inline docs/en/apiserver.md]: please help change this sentence to :
Please refer to this [RFC](https://github.com/4paradigm/rfcs/blob/main/fedb/api-server.md) for the comparison of JSON and OpenMLDB data types.

File: docs/en/apiserver.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                   c5 float,
+                   c6 double,
+                   c7 timestamp,
+                   c8 date,
+                   index(key=c1, ts=c7));
+```
+
+then start APIServer at localhost, port is 8080.
+
+Considering the transmission efficiency，response use Compact JSON format. But for the convenience of display,repsonse below are formatted(Line breaks and indents).
```
Comment (lumianph) [inline docs/en/apiserver.md]: response use
=> the response uses
response below are formatted
=> the response below is formatted

File: docs/en/apiserver.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+request body: 
+```
+{
+    "value": [
+    	[v1, v2, v3]
+    ]
+}
+```
+
+Currently only support insert one record, data arranged in strict accordance with the schema in table.
```
Comment (lumianph) [inline docs/en/apiserver.md]: support "insert" => inserting

File: docs/en/apiserver.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+```
+{
+    "common_cols":["value1", "value2"],
+    "input": [["value0", "value3"],["value0", "value3"]],
+    "need_schema": true
+}
+```
+
++ common_cols could be empty if there is no data.
++ need_schema could be `false` or just empty, then there will have not schema field in response.
```
Comment (lumianph) [inline docs/en/apiserver.md]: then there will have not schema field in response.
=>
then there will be no schema field in a response

**Review by lumianph — CHANGES_REQUESTED** [review]
thank you for the contribution, please help address the comments. Thanks.

File: docs/en/apiserver.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ create table trans(c1 string,
 
 then start APIServer at localhost, port is 8080.
 
-Considering the transmission efficiency，response use Compact JSON format. But for the convenience of display,repsonse below are formatted(Line breaks and indents).
+Considering the transmission efficiency，the response use Compact JSON format. But for the convenience of display, the repsonse below are formatted(Line breaks and indents).
```
Comment (lumianph) [inline docs/en/apiserver.md]: the response "use" => uses
the response below "are" formatted => is

**Review by lumianph — CHANGES_REQUESTED** [review]
please help address the minor comments, thanks.

**Review by lumianph — APPROVED** [review]
LGTM
thank you for the contribution!

**Comment by lumianph** [comment]
@mahengyang  BTW, I found the English version of deployment doc is also missing, which breaks the link in this doc, is it possible you also help this related issue: https://github.com/4paradigm/OpenMLDB/issues/914. 
Thanks a lot! I believe we can double the reward for you :-D

File: docs/en/apiserver.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
+# APIServer
+
+To deploy `APIServer` click this: [Deploy OpenMLDB](deploy.md).
```
Comment (dl239) [inline docs/en/apiserver.md]: `deploy.md` is missing, please delete this line @mahengyang

**Comment by mahengyang** [comment]
@lumianph with my pleasure. if you don't abandon, I will make more contribute to OpenMLDB with my poor english.

**Comment by mahengyang** [comment]
I'm sorry, I made a mistake by click the `draft` link.And got a message like this:
`mahengyang marked this pull request as draft 1 minute ago`

**Comment by lumianph** [comment]
> I'm sorry, I made a mistake by click the `draft` link.And got a message like this: `mahengyang marked this pull request as draft 1 minute ago`

no worry, you can submit your PR again, it can turn to the "Open" status.

**Review by dl239 — APPROVED** [review]

**Comment by lumianph** [comment]
[community pr/issue tracking](https://trello.com/c/S4D8P57j/17-community-pr-issue-tracking)

**Pull request merged**